src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    public static void printList(String[] tasks, int taskCount, int[] taskDone) {
+        String[] currList = Arrays.copyOf(tasks, taskCount + 1);
```

Comment:
can have a more descriptive variable name as people may not know what the list consists of